input/intro-notes/StructureDefinition-au-core-encounter-notes.md

```diff
@@ -171,3 +171,36 @@ The following search parameters and search parameter combinations **SHOULD** be
 
     *Implementation Notes:* Fetches a bundle of all Encounter resources matching the specified patient and location ([how to search by reference](http://hl7.org/fhir/R4/search.html#reference))
 
+1. **SHOULD** support searching using the combination of the **[`patient`](https://hl7.org/fhir/R4/encounter.html#search)** and **[`discharge-disposition`](http://build.fhir.org/ig/hl7au/au-fhir-base/SearchParameter-encounter-discharge-disposition.html)** search parameters:
+    - **SHOULD** support chained searching of patient canonical identifier `patient.identifier` (e.g. `patient.identifier=[system|][code]`)
+
+    `GET [base]/Encounter?patient={Type/}[id]&discharge-disposition={system|}[code]`
+
+    Example:
+    
+      1. GET [base]/Encounter?patient=5678&amp;discharge-disposition=9
+      1. GET [base]/Encounter?patient=5678&amp;discharge-disposition=9&amp;
```

Comment:
9& as the discharge-disposition param value needs to be fixed. This is a copy and paste error and should have the system instead of "&". Recommend fixing the above typo at the same time.

Reply:
Fixed.